Gemfile

```diff
@@ -31,6 +31,10 @@ group :test do
   gem 'mail'
   gem 'et_ccd_client', git: 'https://github.com/hmcts/et-ccd-client-ruby.git', tag: 'v0.4.0'
   gem 'et_test_helpers', git: 'https://github.com/hmcts/et_test_helpers.git', tag: 'v1.1.4'
+
+  # rtf and pdf file reader
+  gem 'ruby-rtf'
+  gem 'pdf-reader'
```

Comment:
Might be worth locking these to a specific version of each gem, that way we don't have issues at a later date if they auto update on a bundle.